addon/components/polaris-badge.js

```diff
@@ -112,23 +106,17 @@ export default class PolarisBadge extends Component {
     return null;
   }
 
-  @computed('status')
   get statusDescription() {
-    const { status } = this;
-    if (isBlank(status) || status === 'default') {
-      return null;
-    }
-
-    return STATUS_LABELS[status];
+    return this.args.status ? STATUS_LABELS[this.args.status] : null;
   }
 
-  @computed('status')
   get statusClass() {
-    const { status } = this;
-    if (isBlank(status) || status === 'default') {
-      return null;
-    }
+    return this.args.status
+      ? `Polaris-Badge--status${classify(this.args.status)}`
+      : null;
+  }
 
-    return `Polaris-Badge--status${classify(status)}`;
+  get withinFilterClass() {
+    return this.withinFilterContext ? 'Polaris-Badge--withinFilter' : null;
```

Comment:
should this be `this.args.withinFilterContext`?

Reply:
This attr uses our `@argWithDefault` decorator with a default of `false`, so we don't need to go via `args` here since the decorator does that for us.

Reply:
oh, yes, pls ignore my comment 🤦